### PR TITLE
HBW-118 Fix select_table behavior in multi mode

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ v1.6.1 [unreleased]
 -   [#245](https://github.com/latera/homs/pull/245) Disable business process buttons when loading.
 -   [#247](https://github.com/latera/homs/pull/247) Use internal ports in homs_network.
 -   [#250](https://github.com/latera/homs/pull/250) Fix file uploading.
+-   [#258](https://github.com/latera/homs/pull/252) Fix `select_table` behavior in multi mode.
```
Comment (sugarfree1) [inline CHANGELOG.md]: It is 252, not 258

Comment (mxsmirnoff) [reply CHANGELOG.md]: Oops

**Comment by mxsmirnoff** [comment]
Updated

**Comment by mxsmirnoff** [comment]
Resolved conflicts

**Pull request merged**